diff --git a/dist/esm/actions/getConnectorClient.js b/dist/esm/actions/getConnectorClient.js
index 5dc2c9997bc58cc45f38b3d27bef2eeddf13237c..e1cb33e8751c957546dd87539a237fa4b230899a 100644
--- a/dist/esm/actions/getConnectorClient.js
+++ b/dist/esm/actions/getConnectorClient.js
@@ -31,12 +31,12 @@ export async function getConnectorClient(config, parameters = {}) {
         throw new ConnectorNotConnectedError();
     const chainId = parameters.chainId ?? connection.chainId;
     // Check connector using same chainId as connection
-    const connectorChainId = await connection.connector.getChainId();
-    if (connectorChainId !== chainId)
-        throw new ConnectorChainMismatchError({
-            connectionChainId: chainId,
-            connectorChainId,
-        });
+    // const connectorChainId = await connection.connector.getChainId();
+    // if (connectorChainId !== chainId)
+    //     throw new ConnectorChainMismatchError({
+    //         connectionChainId: chainId,
+    //         connectorChainId,
+    //     });
     const connector = connection.connector;
     if (connector.getClient)
         return connector.getClient({ chainId });
